modules/telco-ran-node-tuning-operator.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-ref-du-components.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-ran-node-tuning-operator_{context}"]
+= Node Tuning Operator
+
+New in this release::
+* No reference design updates in this release
+
+Description::
+You tune the cluster performance by link:https://docs.openshift.com/container-platform/latest/scalability_and_performance/cnf-create-performance-profiles.html[creating a performance profile].
+Settings that you configure with a performance profile include:
++
+* Selecting the realtime or non-realtime kernel.
+
+* Allocating cores to a reserved or isolated `cpuset`.
+{product-title} processes allocated to the management workload partition are pinned to reserved set.
+
+* Enabling kubelet features (CPU manager, topology manager, and memory manager).
+
+* Configuring huge pages.
+
+* Setting additional kernel arguments.
+
+* Setting per-core power tuning and max CPU frequency.
+
+Limits and requirements::
+
+The Node Tuning Operator uses the `PerformanceProfile` CR to configure the following for DU profile cluster nodes:
+
+* Sets the reserved `cpuset` to 4 or more 2-core hyperthreads on Intel 3rd Generation Xeon (Ice Lake) 2.20 GHz CPUs or better with firmware tuned for maximum performance.
+
+* Sets the reserved `cpuset` to include both hyperthread siblings for each included core. Unreserved cores are available as allocatable CPU for scheduling workloads.
+
+* Configures reserved and isolated CPUs to include all threads of all cores.
+
+* Sets Core 0 of each NUMA node to be included in the reserved CPU set.
+
+* Sets the huge page size to 1G.
+
+* Pins OpenShift pods that are configured by default as part of the management workload partition to the reserved cores.
```

Comment:
The intent behind this statement is that the user should not opt additional workload into the management partition. ie only those pods which are part of OpenShift that we, by default, have annotated into the management partition.